extensions/blocks/conversation/editor.scss

```diff
@@ -0,0 +1,40 @@
+// Settings sidebar.
+$participant-height: 30px;
+
+.components-panel__body.wp-block-jetpack-conversation__participants {
+	.components-base-control {
+		margin-bottom: 0;
+	}
+}
+
+.wp-block-jetpack-conversation__participant {
+	display: flex;
+
+	.components-base-control {
+		margin-bottom: 5px;
+		flex-grow: 2;
+	}
+
+	.components-button {
+		margin-left: 5px;
```

Comment:
The left margin knocks "Add Participant" button out of alignment on the left side

<img width="271" alt="image" src="https://user-images.githubusercontent.com/1699996/101436523-f3888f00-38d3-11eb-8d7d-3346dc84300e.png">

Reply:
removed the margin here https://github.com/Automattic/jetpack/pull/17916/commits/17ce0afc2070e9415b30f03522ab77184c87c7ac